pwndbg/lib/memory.py

```diff
@@ -140,7 +140,13 @@ def permstr(self) -> str:
         )
 
     def __str__(self) -> str:
-        return f"{self.vaddr:#{2 + 2 * pwndbg.aglib.arch.ptrsize}x} {self.vaddr + self.memsz:#{2 + 2 * pwndbg.aglib.arch.ptrsize}x} {self.permstr} {self.memsz:8x} {self.offset:6x} {self.objfile or ''}"
+        from os.path import relpath
+        if pwndbg.config.vmmap_prefer_relpaths:
+            rel = relpath(self.objfile)
+            objfile = self.objfile if rel.startswith('../../') else rel
+        else:
+            objfile = self.objfile
+        return f"{self.vaddr:#{2 + 2 * pwndbg.aglib.arch.ptrsize}x} {self.vaddr + self.memsz:#{2 + 2 * pwndbg.aglib.arch.ptrsize}x} {self.permstr} {self.memsz:8x} {self.offset:6x} {objfile or ''}"
```

Comment:
If we are here, since we are computing `2 + 2 * pwndbg.aglib.arch.ptrsize` twice, can we put this on a variable above and refer that variable here?